transform/transformers/base/marketplace.py

```diff
@@ -0,0 +1,126 @@
+# pylint: disable=line-too-long, wildcard-import, unused-wildcard-import, invalid-name
+"""Transform Marketplace's resources"""
+from abc import abstractmethod
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import (
+    split,
+)
+from pyspark.sql.types import (
+    StructType,
+    StructField,
+    StringType,
+    BooleanType,
+)
+from transform.transformations.common import *
+from transform.transformers.base.base import BaseTransformer
+from transform.utils.utils import sort_schema
+from transform.schemas.properties_name import ID
+
+SERVICE_TYPE = "service"
+DATA_SOURCE_TYPE = "data source"
+
+
+class MarketplaceBaseTransformer(BaseTransformer):
+    """Transformer used to transform Marketplace's resources"""
+
+    def __init__(
+        self,
+        desired_type: str,
+        cols_to_add: Tuple,
+        cols_to_drop: Tuple,
+        spark: SparkSession,
+    ):
+        super().__init__(
+            desired_type, cols_to_add, cols_to_drop, self.cols_to_rename, spark
+        )
+
+    def apply_simple_trans(self, df: DataFrame) -> DataFrame:
+        """Apply simple transformations.
+        Simple in a way that there is a possibility to manipulate the main dataframe
+        without a need to create another dataframe and merging"""
+        df = df.withColumn(TYPE, lit(self.type))
+        df = self.rename_cols(df)
+        df = self.simplify_urls(df)
+        # Increase the range of data sources IDs -> to avoid a conflict with service IDs
+        if self.type == DATA_SOURCE_TYPE:
+            df = df.withColumn(ID, (col(ID) + 10_000_000))
```

Comment:
This should be extracted to constant - otherwise it becomes magic number.

Maybe even it might be wrapped into separate function `map_id(df) -> str`, or something.